src/Bundle/DependencyInjection/Compiler/RegisterControllerAliasesPass.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\DependencyInjection\Compiler;
+
+use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
+use Sylius\Component\Resource\Metadata\Metadata;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Symfony\Component\DependencyInjection\Alias;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+
+final class RegisterControllerAliasesPass implements CompilerPassInterface
```

Comment:
We should adjust the name, as this class is not aliasing controllers anymore
```suggestion
final class RegisterFQCNControllersPass implements CompilerPassInterface
```

Reply:
Lets maybe call it `@internal`, so we can adjust its name in the future
```suggestion
@internal
final class RegisterControllerAliasesPass implements CompilerPassInterface
```